connector/mqtt/vernemq/broker/src/dojot_disconnect_plugin/readme.md

```diff
@@ -1,37 +1,41 @@
 # Dojot Disconnect Plugin
 
-This plugin checks and closes staled connections for dojot. The important files are:
+This plugin disconnects clients from VerneMQ after some time. This increases the security because
```

Comment:
can you explain more or add a reference about the PLUGIN_DISC_LIFETIME_SESSION  env variable?